tests/checkPot.py

```diff
@@ -0,0 +1,192 @@
+	'header item',
+	'calendar',
+	'video',
+	'audio',
+	'modal',
+	'iconified',
+	'editable',
+	'checkable',
+	'draggable',
+	'dragging',
+	'sorted',
+	'sorted ascending',
+	'sorted descending',
+	'gesture map File Error',
+	'text \\"%s\\" not found',
+	'Find Error',
+	'Selected %s',
+	'on',
+	'Type help(object) to get help about object.',
+	'Type exit() to exit the console',
+	'NVDA Python Console',
+	'Emulates pressing %s on the system keyboard',
+	'continuous section break',
+	'new column section break',
+	'new page section break',
+	'even pages section break',
+	'odd pages section break',
+	'column break',
+	'background pattern {pattern}',
+	'NVDA Speech Viewer',
+	'text mode',
+	'object mode',
+	'NonVisual Desktop Access',
+	'A free and open source screen reader for Microsoft Windows',
+	'Copyright (C) {years} NVDA Contributors',
+	'Display',
+	'Reports and moves the review cursor to a recent message',
+	'left',
+	'right',
+	'Show the Handy Tech driver configuration window.',
+	'Recognition failed',
+	'General settings',
+	'Change the synthesizer to be used',
+	'Choose the voice, rate, pitch and volume to use',
+	'Change reporting of mouse shape and object under mouse',
+	'Configure how and when the review cursor moves',
+	'Change reporting of objects',
+	'Change virtual buffers specific settings',
+	'Change settings of document properties',
+	'NVDA &web site',
+	'Reset all settings to saved state',
+	'Reset all settings to default state',
+	'Write the current configuration to nvda.ini',
+	'E&xit',
+	'Options',
+	'Error renaming profile.',
+	'Use this profile for:',
+	'This change requires administrator privileges.',
+	'Insufficient Privileges',
+	'Synthesizer Error',
+	'Dictionary Entry Error',
+	'Could not load the %s display.',
+	'Braille Display Error',
+	'word',
+	'Taskbar',
+	'%s items',
+	'invoke',
+	'Desktop',
+	'Input Message is {title}: {message}',
+	'Input Message is {message}',
+	'Series color: {colorName} ',
+	'Comments',
+	'Endnotes',
+	'Even pages footer',
+	'Even pages header',
+	'First page footer',
+	'First page header',
+	'Footnotes',
+	'Primary footer',
+	'Primary header',
+	'Text frame',
+}
+
+def checkPot(fileName):
+	"""Returns the number of errors.
+	Also prints error messages and a summary to standard output.
+	"""
+	errors = 0
+	expectedErrors = 0
+	unexpectedSuccesses = 0
+	with file(fileName, "rt") as pot:
+		for line in pot:
+			line = line.rstrip()
+			if not line:
+				# End of message.
+				hasComment = False
+				sourceLines = []
+				context = ""
+				continue
+			if line == 'msgid ""':
+				# Header.
+				continue
+			if line.startswith("#. Translators: "):
+				# Example: "#. Translators: a message reported in the SetRowHeader script for Microsoft Word."
+				hasComment = True
+				continue
+			if line.startswith("#: "):
+				# Example: "#: NVDAObjects\window\winword.py:1322"
+				# Strip the "#: " prefix (3 chars).
+				sourceLines.append(line[3:])
+				continue
+			if line.startswith('msgctxt "'):

```

Comment:
It's probably assumed knowledge for this, but what are: `msgctxt`, `msgid`, `msgstr`. Are these concepts from some other library module? Either a brief explanation of what they are or a pointer to where to find out about them could be handy for someone who comes to maintain this without that background knowledge.